### DISCUSSION TITLE
Casa OS container

### DISCUSSION THREAD

**Discussion by jtrofficial** [DustinBrett/daedalOS#462] [Feature Requests]
It would be nice if you could add Daedal OS to the CasaOS app store; [CasaOS is a "A simple, easy-to-use, elegant open-source Personal Cloud system." ](https://github.com/IceWhaleTech/CasaOS)  CasaOS uses Docker containers as "Apps" or webservices hosted in CasaOS (Note: CasaOS is not a Operating system; it is a service you run on almost any Linux DIstro.) Adding a download for a docker container; or even better getting it added to the Casa Web store would allow more people to host, use, and test Daedal OS without any setup Except Running the one command to install CasaOS (curl -fsSL https://get.casaos.io | sudo bash) and then loading up their local ip address at port 80 and navigating the gui and clicking a few buttons instead of several terminal commands that non-tech acquainted people may not be able to grasp how to do.
![Screenshot 2024-07-25 005554](https://github.com/user-attachments/assets/7303fbcb-650e-4458-af8a-2ef59c414a2a)
![Screenshot 2024-07-25 005609](https://github.com/user-attachments/assets/6604afb1-304a-41eb-8b11-2833e5282c8b)

**Comment by DustinBrett** [comment]
It's a cool idea for sure. I think it's for sure possible, but I hadn't heard of CasaOS before. Maybe one day they will add it, or perhaps I will eventually. I still am keeping at making improvements. Thanks for the suggestion!
